go/cmd/vtaclcheck/vtaclcheck.go

```diff
@@ -17,44 +17,26 @@ limitations under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
 
+	"github.com/spf13/pflag"
+
 	"vitess.io/vitess/go/exit"
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vt/servenv"
 	"vitess.io/vitess/go/vt/vtaclcheck"
-
-	// Include deprecation warnings for soon-to-be-unsupported flag invocations.
-	_flag "vitess.io/vitess/go/internal/flag"
 )
 
-var (
-	aclFileFlag        = flag.String("acl_file", "", "The path of the JSON ACL file to check")
-	staticAuthFileFlag = flag.String("static_auth_file", "", "The path of the auth_server_static JSON file to check")
-
-	// vtaclcheckFlags lists all the flags that should show in usage
-	vtaclcheckFlags = []string{
-		"acl_file",
-		"static_auth_file",
-	}
-)
+var aclFile, staticAuthFile string
 
 func init() {
 	logger := logutil.NewConsoleLogger()
-	flag.CommandLine.SetOutput(logutil.NewLoggerWriter(logger))
-	_flag.SetUsage(flag.CommandLine, _flag.UsageOptions{
-		FlagFilter: func(f *flag.Flag) bool {
-			for _, name := range vtaclcheckFlags {
-				if f.Name == name {
-					return true
-				}
-			}
-
-			return false
-		},
+	servenv.OnParse(func(fs *pflag.FlagSet) {
+		fs.StringVar(&aclFile, "acl-file", aclFile, "The path of the JSON ACL file to check")
+		fs.StringVar(&staticAuthFile, "static-auth-file", staticAuthFile, "The path of the auth_server_static JSON file to check")
 	})
+	pflag.CommandLine.SetOutput(logutil.NewLoggerWriter(logger))
```

Comment:
```suggestion
	servenv.OnParse(func(fs *pflag.FlagSet) {
		fs.StringVar(&aclFile, "acl-file", aclFile, "The path of the JSON ACL file to check")
		fs.StringVar(&staticAuthFile, "static-auth-file", staticAuthFile, "The path of the auth_server_static JSON file to check")
		
		fs.SetOutput(logutil.NewLoggerWriter(logger))
	})
```

this needs to be done to the `fs` used by `servenv` (which you have access to only in the callback function above)